include/aoscene.h

```diff
@@ -23,11 +54,37 @@ class AOScene : public QLabel {
   // no "combo" to speak of
   void combo_resize(int w, int h);
 
+  // Return the frame delay adjusted for speed
+  int get_frame_delay(int delay);
+
+  QStringList network_strings;
+
+  QString m_char;
+  QString m_emote;
```

Comment:
public member fields - should these be private?

Reply:
m_char and m_emote should be private, however network_strings is accessed elsewhere and must remain public